sky/jobs/utils.py

```diff
@@ -948,13 +953,13 @@ def format_job_table(
       a list of "rows" (each of which is a list of str).
     """
     jobs = collections.defaultdict(list)
-    # Check if the tasks have user information.
-    tasks_have_user = any([task.get('user') for task in tasks])
-    if max_jobs and tasks_have_user:
+    # Check if the tasks have user information from kubernetes.
+    tasks_have_k8s_user = any([task.get('user') for task in tasks])
```

Comment:
Add a comment saying this is only used in `sky status --kubernetes`.

Also, we should consider getting rid of the `sky status --kubernetes` now, since people should be able to see a global view with the deployed API server now.